solhint.js

```diff
@@ -128,13 +132,30 @@ const readConfig = _.memoize(() => {
   let config = {}
 
   try {
-    const configStr = fs.readFileSync('.solhint.json').toString()
+    let configStr
```

Comment:
I think we should do:

```js
const configStr = program.configPath || '.solhint.json'
```

and then adapt the following lines accordingly.

Reply:
Solved!